pages/mesosphere/dcos/services/dse/3.1.0-6.7.6/updates/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: layout.pug
+navigationTitle: Updating 
+excerpt: Updating DSE
+title: Updating 
+menuWeight: 40
+model: /mesosphere/dcos/services/dse/data.yml
+render: mustache
+---
+
+<!-- #include /mesosphere/dcos/services/include/update.tmpl -->
+
+# Upgrading your cluster from {{ model.techShortName }} 5.1.10 to 6.7.6
+Due to the complexity of upgrading to {{ model.techShortName }} 6.7, we strongly advise that you attempt the upgrade on a test cluster before upgrading in your production environment. 
+
+Also we advise that you upgrade 5.1.10 directly to 6.7.6, skipping 6.7.2, as it has a problem in upgrading when TLS is enabled. 
+
+<p class="message--important"><strong>IMPORTANT: </strong> 
+<ul>
+<li>This action cannot be undone and you should plan for increased load activity on your cluster. This task should be scheduled for off-peak hours. Should any problems arise, <tt>pause</tt> the plan and investigate.</li>
+<li>Value of one of the properties <tt>dse_search</tt> or <tt>dse_analytics</tt> should be set to <tt>true</tt> before upgrading due to some limitation, later it can be set as needed. </li>
+<li>Upgrade will not work on DC/OS 2.0.0 becasue of a bug, please use latest DC/OS 2.x.x instead.</li>
```

Comment:
> becasue of a bug

because of what bug? This is very ambiguous to a user

>  please use latest DC/OS 2.x.x instead.</li>

what version should they use? Do we have a DCOS 2.0.1 yet? This message is not clear to a user. If the upgrade doesn't work on DCOS 2.0.0 yet, then I'd suggest simply stating that. And when we release a version that it _does_ work on, then come back and update this doc.

Reply:
ok should I mention about the bug?
yes we have DCOS 2.0.1.

Reply:
```suggestion
<li>Due to a bug in DC/OS 2.0.0, upgrading DSE to 6.7.6 will not work. Instead, please upgrade DC/OS to 2.0.1 and then upgrade DSE to 6.7.6.</li>
```
Okay how about this?